appbuildsystem/ansible/appbuild.py

```
@@ -115,6 +110,18 @@
 else:
     error("unsupported OS!")
 
+# name: install virtualenv
+py_version = get_virtualenv_python_version(virtenv_path, host_post_info)
```

Ruff (0.15.2)

[error] `get_virtualenv_python_version` may be undefined, or defined from star imports

(F405)